orttraining/orttraining/python/training/onnxblock/optim/optim.py

```diff
@@ -1,30 +1,24 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
-# optim.py
 
-import typing
+from typing import Optional, Tuple
 
 import onnx
 
-import onnxruntime.training.onnxblock._graph_utils as graph_utils
-import onnxruntime.training.onnxblock.building_blocks as building_blocks
-import onnxruntime.training.onnxblock.model as model
-import onnxruntime.training.onnxblock.model_accessor as accessor
+import onnxruntime.training.onnxblock._graph_utils as _graph_utils
```

Comment:
```suggestion
from onnxruntime.training.onnxblock import _graph_utils
```

consider this. here and elsewhere?

Reply:
Is there an advantage in doing it this way.

My thought process is to distinguish imports from __init__.py and local files with this way of importing and is purely subjective.

I don't know if there is any advantage of doing one way over the other.

Reply:
Just to avoid repetition. NBD. I think we should use as only when renaming.

Reply:
I'll let this be as is since I would like to avoid having to type out `onnxruntime.training.onnxblock._graph_utils` over and over again.

If you feel strongly about this, I will go ahead and make the change.

Reply:
Just making sure, I think `from onnxruntime.training.onnxblock import _graph_utils` will save you from typing the whole thing too?

Reply:
Spoke to Justin offline. Will keep this as is for now. Thanks, @justinchuby